src/lib/rest/RestService.h

```diff
@@ -91,4 +82,33 @@ extern std::string payloadParse
 */
 extern std::string tenantCheck(const std::string& tenant);
 
+
+
+/* ****************************************************************************
+*
+* serviceVectorsSet - only for unit tests
```

Comment:
It seems that at the end this function is not only used only for unit tests, as restInit() also uses it. Thus the "- only for unit tests" fragment should be removed.

Reply:
(Same in the .ccp file)

Reply:
Fixed in a265c792e